stetho/src/main/java/com/facebook/stetho/inspector/protocol/module/Database.java

```diff
@@ -46,36 +45,46 @@
    */
   private static final int MAX_EXECUTE_RESULTS = 250;
 
-  private final DatabasePeerManager mDatabasePeerManager;
+  private List<DatabaseDriver> mDatabaseDrivers;
+  private final ChromePeerManager mChromePeerManager;
   private final ObjectMapper mObjectMapper;
 
   /**
-   * Constructs the object with the default {@link DatabasePeerManager}.
-   * @param context the context
+   * Constructs the object.
    */
-  @Deprecated
-  public Database(Context context) {
-    mDatabasePeerManager = new DatabasePeerManager(context);
+  public Database() {
+    mDatabaseDrivers = new ArrayList<>();
+    mChromePeerManager = new ChromePeerManager();
+    mChromePeerManager.setListener(new PeerRegistrationListener() {
```

Comment:
nit, prefer this in a named subclass or a member variable for improved fluency:

    mChromePeerManager.setListener(mPeerListener);